packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -147,12 +147,9 @@ describe('plugin-conversation', () => {
 
     describe('paginate', () => {
       it('should throw an error if a page is passed with no links', () => {
-        try {
-          webex.internal.conversation.paginate({page: {}});
-        }
-        catch (error) {
+        webex.internal.conversation.paginate({page: {}}).catch((error) => {
```

Comment:
"We should add a fail() call after the call to paginate() so that the test fails if paginate doesn't throw. Or even better we should just use toThrow() to check that a function throws."
@marcin-benzyl 
Paginate will always throw if we don't pass page links. It should throw as this is what we're testing for this method as paginate method is written this way only. Check below code reference:
`async paginate(options = {}) {
    if (options.page) {
      // We were passed a page but we are out of results
      if (!options.page.links || !options.page.links.next) {
        throw new Error('No link to follow for the provided page');
      }`